manifests/plugin/foreman_gutterball.pp

```diff
@@ -1,12 +1,18 @@
 # foreman_gutterball plugin
 class katello_devel::plugin::foreman_gutterball {
   $path = "${katello_devel::deployment_dir}/foreman-gutterball"
+  $foreman_settings_path = "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d"
```

Comment:
@komidore64 wondering if we need the config file from foreman-gutterball or just keep all the configs here, in puppet land?